scripts/jira/cloner.py

```diff
@@ -277,48 +317,48 @@ def query_build(issue, user):
 
     args = parser.parse_args()
 
-    connection = jira.JIRA(
+    conn = jira.JIRA(
         server=JIRA_SERVER,
         token_auth=args.token)
 
     jql_query = query_build(args.issue, args.user)
     print(f"JQL Query: '{jql_query}'")
 
     try:
-        query = connection.search_issues(jql_str=jql_query, maxResults=9999)
+        query = conn.search_issues(jql_str=jql_query, maxResults=9999)
         print(f"Scanning {len(query)} issues")
     except Exception as e:
         print(f"Unable to retrieve issues: {e}")
-        exit(1)
+        sys.exit(1)
 
-    actions = []
+    actions_list = []
     for i in tqdm(range(len(query))):
         try:
-            issue = query[i]
+            issuee = query[i]
```

Comment:
What is reason for this change?

Reply:
`W0621: Redefining name 'issue' from outer scope (line 338) (redefined-outer-name)`

Because `issue` is getting used in most of the function definitions. So, either change it in `main` function or in all the functions where `issue` is used. I chose to change in `main` function for simplicity and less line of change.